examples/finance/bs_ivbp.ipynb

```diff
@@ -37,13 +37,12 @@
    ],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #65.    print("dt,tmax,nt;", dt0,tmax,nt)
</pre><p>These should use fstrings imo</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/2542/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>